docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,265 @@
+  access to organizational hardware, data, and analytics environments at scale.
+
+Here are a sample of organizations that use JupyterHub:
+
+* **Universities and colleges**: UC Berkeley, UC San Diego, Cal Poly SLO, Harvard University, University of Chicago,
+  University of Oslo, University of Sheffield, Université Paris Sud, University of Versailles
+* **Research laboratories**: NASA, NCAR, NOAA, the Large Synoptic Survey Telescope, Brookhaven National Lab,
+  Minnesota Supercomputing Institute, ALCF, CERN, Lawrence Livermore National Laboratory
+* **Online communities**: Pangeo, Quantopian, mybinder.org, MathHub, Open Humans
+* **Computing infrastructure providers**: NERSC, San Diego Supercomputing Center, Compute Canada
+* **Companies**: Capital One, SANDVIK code, Globus
+
+See the [Gallery of JupyterHub deployments](../gallery-jhub-deployments.md) for
+a more complete list of JupyterHub deployments at institutions.
+
+## How does JupyterHub compare with hosted products, like Google Colaboratory, RStudio.cloud, or Anaconda Enterprise?
+
+JupyterHub puts you in control of your data, infrastructure, and coding environment.
+In addition, it is vendor neutral, which reduces lock-in to a particular vendor or service.
+JupyterHub provides access to interactive computing environments in the cloud (similar to each of these services).
+Compared with the tools above, it is more flexible, more customizable, free, and
+gives administrators more control over their setup and hardware.
+
+Because JupyterHub is an open-source, community-driven tool, it can be extended and
+modified to fit an institution's needs. It plays nicely with the open source data science
+stack, and can serve a variety of computing enviroments, user interfaces, and
+computational hardware. It can also be deployed anywhere - on enterprise cloud infrastructure, on
+High-Performance-Computing machines, on local hardware, or even on a single laptop, which
+is not possible with most other tools for shared interactive computing.
+
+# For IT
+
+## How would I set up JupyterHub on institutional hardware?
+
+That depends on what kind of hardware you've got. JupyterHub is flexible enough to be deployed
+on a variety of hardware, including in-room hardware, on-prem clusters, cloud infrastructure,
+etc.
+
+The most common way to set up a JupyterHub us to use a JupyterHub distribution, these are pre-configured
+and opinionated ways to set up a JupyterHub on particular kinds of infrastructure. The two distributions
+that we currently suggest are:
+
+* [Zero to JupyterHub for Kubernetes](https://z2jh.jupyter.org) is a scalable JupyterHub deployment and
+  guide that runs on Kubernetes. Better for larger or dynamic user groups (50-10,000) or more complex
+  compute/data needs.
+* [The Littlest JupyterHub](https://tljh.jupyter.org) is a lightweight JupyterHub that runs on a single
+  VM in the cloud. Better for smaller usergroups (4-80) or more lightweight computational resources.
```

Comment:
replace "single VM in the cloud" with "single machine (in the cloud or under your desk)"?